api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -516,6 +523,43 @@ message HttpConnectionManager {
   bool strip_matching_host_port = 39;
 }
 
+// The configuration to customize local reply returned by Envoy.
+message LocalReplyConfig {
+  // Configuration of list of mappers which allows to filter and change local response.
+  // The mappers will be checked by the specified order until one is matched.
+  repeated ResponseMapper mappers = 1;
+
+  // The configuration to form response body from the :ref:`command operators <config_access_log_command_operators>`
+  // and to specify response content type as one of: plain/text or application/json.
+  // If the matched `mapper <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.LocalReplyConfig.mapper>`
+  // has a valid `format <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.ResponseMapper.format>`,
+  // it will be used, this one will not be used.
+  config.core.v3.SubstitutionFormatString format = 2;
```

Comment:
Maybe body_format in case we add control of headers or something else in the future? Same below?